src/maheshKJun20/CharTypeBreakup.java

```diff
@@ -0,0 +1,31 @@
+package maheshKJun20;
+
+public class CharTypeBreakup {
+	
+	void checkCharType(String str) {
+		//Method will find the number of Digits, Letters, Special Character, Uppercase letter and lowercase letter from the input string
+		System.out.println("Input String value: " + str);
+		int countDigit = 0, countLetters =0, countUppercase =0, countLowercase = 0, countSpecialChar =0;
+		for (int index=0;index<str.length();index++) {			if (Character.isDigit(str.charAt(index))) {
+				countDigit++;
+			}else if (Character.isLetter(str.charAt(index))) {
+				countLetters++;
+				if (Character.isUpperCase(str.charAt(index))) {
+					countUppercase++;
+				}else {
```

Comment:
Try to implement logic without using countLowercase variable and else if (Character.isLetter(str.charAt(index))) block. Because both are not required to achive expected output.